pygeos/constructive.py

```diff
@@ -109,7 +109,7 @@ def buffer(
     --------
     >>> buffer(Geometry("POINT (10 10)"), 2, quadsegs=1)
     <pygeos.Geometry POLYGON ((12 10, 10 8, 8 10, 10 12, 12 10))>
-    >>> buffer(Geometry("POINT (10 10)"), 2, quadsegs=2)
+    >>> buffer(Geometry("POINT (10 10)"), 2, quadsegs=2))  # doctest: +SKIP
```

Comment:
It might be OK to leave these either by wrapping with `to_wkt()` to print full string (though it does distract a tiny bit from the example)
or include the truncated version of the output, which may be sufficient to demonstrate that it producted a polygon buffer:
```
<pygeos.Geometry POLYGON ((12 10, 11.4 8.59, 10 8, 8.59 8.59, 8 10, 8.59 11....>
```

Reply:
I would also be fine with actually having the truncated one in the docstrings below (it also makes the docstring line length shorter)